test/parallel/test-buffer-alloc.js

```diff
@@ -956,7 +956,7 @@ assert.throws(() => {
   const a = Buffer.alloc(1);
   const b = Buffer.alloc(1);
   a.copy(b, 0, 0x100000000, 0x100000001);
-}, /out of range index/);
+}, /^RangeError(?:\[ERR_INDEX_OUT_OF_RANGE\])?: Index out of range$/);
```

Comment:
This regexp can be replaced with `common.expectsError(undefined, RangeError, 'Index out of range')`